Formula/apk-patch-size-estimator.rb

```diff
@@ -0,0 +1,19 @@
+class ApkPatchSizeEstimator < Formula
+  desc "Estimates the size of Play patches / gzipped APKs."
+  homepage "https://github.com/googlesamples/apk-patch-size-estimator"
+  url "https://github.com/googlesamples/apk-patch-size-estimator/archive/0.2.tar.gz"
+  sha256 "cbab54effbbbb2abf3ddb58cf004e8ee0e38b520fbc5266f08a5fd6a8cf9c9ff"
+  depends_on "bsdiff"
+
+  def install
+    libexec.install Dir["*"]
+
+    Dir.glob("#{libexec}/*.py") do |script|
+      bin.install_symlink script => File.basename(script, ".py").tr("_", "-")
+    end
+  end
+
+  test do
+    assert_match /^usage: apk-patch-size-estimator/, shell_output("#{bin}/apk-patch-size-estimator --help").strip
```

Comment:
I can't seem to find any APKs that are hosted by Google anywhere (besides the Play Store). If you know of one, I'd be glad to update though.